pkg/kubelet/kubelet.go

```diff
@@ -2182,13 +2182,13 @@ func (kl *Kubelet) updateRuntimeUp() {
 		// Set nil if the container runtime network is ready.
 		kl.runtimeState.setNetworkState(nil)
 	}
-	// TODO(random-liu): Add runtime error in runtimeState, and update it
-	// when runtime is not ready, so that the information in RuntimeReady
-	// condition will be propagated to NodeReady condition.
+	// information in RuntimeReady condition will be propagated to NodeReady condition.
```

Comment:
In order for the propagation to occur, do we need to configure some entity to actually read the value of `runtimeError`?

Reply:
+1

Reply:
Updated the PR.
runtimeError is considered by runtimeState#runtimeErrors